docs/source/Plugin/P003_commands.repl

```diff
@@ -3,33 +3,55 @@
         :widths: 20, 30
 
         "
-        ``ResetPulseCounter,<taskIndex>``
+        ``ResetPulseCounter``
+
+        legacy method:
+ 
+        ``ResetPulseCounter,<taskNumber>``
+
         ","
         Reset the counters (pulse counter and total counter) of P003 Pulse Counter.
         
-        The taskIndex parameter is optional.  When not given, only the first task with this plugin will be cleared.
+        When using multiple instances of Pulse Counter tasks, use the the taskNumber as prefix. E.g. ``2.ResetPulseCounter`` addresses task number 2.
+        When taskNumber is neither given as prefix nor as parameter, the first active Pulse Counter task will be affected.
 
-        A call to reset the value will also trigger a call to ``PLUGIN_READ`` as if it processes a normal successful read.
+        The command will also trigger a call to ``PLUGIN_READ`` as if it processes a normal successful read.
         "
         "
-        ``SetPulseCounterTotal,value,<taskIndex>``
+        ``SetPulseCounterTotal,value``
+
+        legacy method:   
+
+        ``SetPulseCounterTotal,value,<taskNumber>``
+
         ","
         Set the value of the total count of P003 Pulse Counter.
         
-        The taskIndex parameter is optional.  When not given, only the total count of the first task with this plugin will be set.
+        When using multiple instances of Pulse Counter tasks, use the the taskNumber as prefix. E.g. ``2.SetPulseCounter,1000`` sets the value in task number 2.
+        When taskNumber is neither given as prefix nor as parameter, the first active Pulse Counter task will be affected.
         
         A call to set the counter total will also trigger a call to ``PLUGIN_READ`` as if it processes a normal successful read.
 
         N.B. The set value is the internal value, so any present formula will be processed after this value is set.
         "
         "
-        ``LogPulseStatistic,<cmd>,<taskindex>``
+        ``LogPulseStatistic,<subcommand>``
+
+        Optional subcommands:
+
+        * ``r`` reset error and overdue counters.
+
+        * ``i`` temporary increases the log level for regular statisticel log entries from ``Debug`` to ``Info``
```

Comment:
The code seems to suggest that subcommand `ri` is also accepted, but it isn't mentioned here?

Reply:
Ah, you removed that feature. Ok, comment wirhdrawn 😄